recipes-bsp/u-boot/files/rpi-3b-disable-bt-fix.patch

```diff
@@ -0,0 +1,29 @@
+--- git/arch/arm/dts/bcm2837-rpi-3-b.dts	2020-11-12 14:32:51.337943522 +0200
```

Comment:
Why do we need to disable BT?

Reply:
To enable the serial connection from GPIO-pins: https://www.raspberrypi.org/documentation/configuration/uart.md

Reply:
Ah, super crappy UARTs again (and not enough of those). Thanks!